test/ClangImporter/objc_parse.swift

```diff
@@ -68,7 +68,7 @@ func classMethods(_ b: B, other: NSObject) {
   i += B.classMethod(1)
   i += B.classMethod(1, with: 2)
 
-  i += b.classMethod() // expected-error{{static member 'classMethod' cannot be used on instance of type 'B'}}
+  i += b.classMethod() // expected-error{{member 'classMethod' cannot be used on value of type 'B'}}
```

Comment:
Can you figure out why `classMethod` no longer diagnosed as `static member` here?

Reply:
@xedin Fixed!